posts/Ippon-USA-offers-a-helping-hand-to-those-impacted-by-natural-disasters.md

```diff
@@ -0,0 +1,32 @@
+ 
+Ippon USA offers a helping hand to those impacted by natural disasters. 
```

Comment:
You don't need to include the title, it will be automatically added from the meta data.